<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MemoryView</class>
 <widget class="QWidget" name="MemoryView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>529</width>
    <height>230</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>4</number>
   </property>
   <property name="topMargin">
    <number>4</number>
   </property>
   <property name="rightMargin">
    <number>4</number>
   </property>
   <property name="bottomMargin">
    <number>4</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="m_Address">
       <property name="placeholderText">
        <string>Type an address expression</string>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="m_Count">
       <property name="editable">
        <bool>true</bool>
       </property>
       <property name="currentText">
        <string>8</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="m_Type">
       <property name="currentIndex">
        <number>-1</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="m_Endianess"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="prodbg::MemoryViewWidget" name="m_View" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>prodbg::MemoryViewWidget</class>
   <extends>QWidget</extends>
   <header>src/prodbg/MemoryView/MemoryViewWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
